### DISCUSSION TITLE
Can't hide the delete action from show page

### DISCUSSION THREAD

**Discussion by yash2411** [activeadmin/activeadmin#7949] [Help]
I am trying to hide the delete action but not able to hide the action from the show page, if i try to hide the action from show page it is also removing it from index. when it try i to hide from show page using this code then 
actions :all, :except => { :show => [ :destroy] } then getting this error:-
ndefined method `to_sym' for [:show, :destroy]:Array (NoMethodError)
Did you mean?  to_s

**Comment by javierjulio** [comment]
Use `actions :all, :except: [:destroy]` as that's the correct format. For the index page, deleting is controlled with batch actions.

**Reply by yash2411** [reply]
But it removes delete link from index page also i just want to remove it from show page

**Reply by javierjulio** [reply]
Sorry, I don't understand. For an index page, deletion is only done through batch actions which is separate from actions.

**Reply by yash2411** [reply]
Could you please explain me with an example?
